red-hat-data-services/odh-dashboard
# [UPSTREAM] RHODS 3159 - PVC and cluster settings page updates

- [x] JIRA link(s): https://issues.redhat.com/browse/RHODS-3159 


This PR adds the ability for a RHODS admin to change notebook image size.

Testing instructions:

RHODS admin testing:

Login to the RHODS dashboard as a RHODS admin.
Verify that Settings in the left hand navigation is present.
Verify that Cluster Settings is a menu item under Settings.
Navigate to Cluster Settings.
Verify that the PVC Size is set to the default size or 20 GiB.
Enter a value in the number input between 1 - 16384 GiB.  Hit Enter to save the image size. (NOTE: This change will cause juypterhub to restart.  It will take 30 seconds before juypterhub will be available.)
After 30 seconds go to Enabled applications launch JuypterHub.
Set up and start your notebook.
In the openshift web console verify the size note book image size is the size of the image value you set under cluster settings
<img width="1298" alt="Screen Shot 2022-03-10 at 8 08 02 PM" src="https://user-images.githubusercontent.com/6126356/157782833-768bc267-2fdf-465a-8f2a-f83fa3ba0c16.png">
.
